lib/acpi/acpi_shell.c

```diff
@@ -267,8 +272,10 @@ SHELL_STATIC_SUBCMD_SET_CREATE(
 	SHELL_CMD(prs, NULL,
 		  "display device possible resource settings (eg:acpi crs _SB.PC00.LPCB.RTC)",
 		  dump_dev_prs),
+#ifdef CONFIG_PCIE_PRT
```

Comment:
I wonder can we use `SHELL_COND_CMD_ARG`

Reply:
> I wonder can we use `SHELL_COND_CMD_ARG`

Yes, that sounds like a good idea

Reply:
> > I wonder can we use `SHELL_COND_CMD_ARG`
> 
> Yes, that sounds like a good idea

Done

Reply:
> > > I wonder can we use `SHELL_COND_CMD_ARG`
> > 
> > 
> > Yes, that sounds like a good idea
> 
> Done

@finikorg this caused build failures, since `SHELL_COND_CMD()` still requires the the callback to defined (not behind ifdef) and therefore any code in the callback to be buildable. I fixed this by moving the `pci_prt_table` array into the callback, and by putting most of the callback implementation behind `IF_ENABLED(...`. CI should hopefully pass now.

Reply:
OK, noticed myself that that CMD_COND requires that workaround